Angular/tsconfig.app.json

```diff
@@ -0,0 +1,15 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./out-tsc/app",
+    "types": []
+  },
+  "files": [
+    "src/main.ts",
+    "src/polyfills.ts"
```

Comment:
Infrastructure comment.

If we will do Infrastructure comments, let's remove polyfills from the tsconfig:

```suggestion
```